metadata_service/api/table.py

```diff
@@ -31,13 +32,37 @@ def __init__(self) -> None:
     def get(self, table_uri: str) -> Iterable[Union[Mapping, int, None]]:
         try:
             table = self.client.get_table(table_uri=table_uri)
-            schema = TableSchema(strict=True)
-            return schema.dump(table).data, HTTPStatus.OK
+            schema = TableSchema()
```

Comment:
I forgot about the context, but why do we remove the strict=true?

Reply:
Similar reason as removing data this wasn't compatible with the update

Reply:
https://marshmallow.readthedocs.io/en/stable/upgrading.html#schemas-are-always-strict